src/components/partials/seoSocial.js

```diff
@@ -31,7 +31,12 @@ const SeoSocial = (props) => {
   return (
     <SbEditable content={props.blok}>
       <Helmet titleTemplate={`%s | ${title}`} title={props.blok.title}>
-        <link rel="canonical" href={canonicalUrl} />
+        {props.blok.noindex ? (
+          <meta name="robots" content="noindex" />
+        ) : (
+          <link rel="canonical" href={canonicalUrl} />
+        )}
```

Comment:
Why the or condition here? What relationship does noindex have with the canonical URL?

Reply:
https://www.searchenginejournal.com/google-dont-mix-noindex-relcanonical/262607/

Reply:
GTK. Thanks for the link.

Reply:
That was a new one for me as well